### DISCUSSION TITLE
No indentation on MenuItem if it's at the top level

### DISCUSSION THREAD

**Discussion by seabasss** [laravel/nova-issues#4187] [Ideas & Feature Requests]
Would it be possible to remove the indentation on `MenuItem` if it's at the top level of the side menu?
A plus would be if `MenuItem` also had an `icon ` method.

Thank you!

**Comment by matthewjumpsoffbuildings** [comment]
Was about to make an issue for this exact thing, both the indentation and the icon.

Since there is now no easy way to override the menu via blade templates, the bare minimum would be _some_ control over the indentation of MenuItem/Group/Section objects. Honestly a _lot_ of the issue is this element
```html
<span class="sidebar-item-icon"></span>
```
that sits inside every Menu object wether it has an icon or not. Just having a `MenuItem->icon` boolean property for all Menu objects that hides that element would make a _world_ of difference

But it would also be very good to add some additional control over indentation, perhaps something like `MenuItem::indentationLevel(1)`
